### PR TITLE
Add horizon-operator deploy and kuttl testing

### PR DESCRIPTION
**Pull request by jparoly**

### CODE REVIEW

**Comment by jparoly** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-horizon-operator-main-horizon-operator-build-deploy-kuttl

File: ci-operator/config/openstack-k8s-operators/horizon-operator/openstack-k8s-operators-horizon-operator-main.yaml [hunk 1]
```diff
@@ -6,6 +6,18 @@ images:
...
+    value: v0.15.0
+  dockerfile_literal: |
+    FROM openstack-operator-ci-build-deploy
+    ARG KUTTL_VERSION=v0.15.0
+    RUN curl -s -L "https://github.com/kudobuilder/kuttl/releases/download/${KUTTL_VERSION}/kubectl-kuttl_${KUTTL_VERSION#v}_linux_x86_64" -o kubectl-kuttl
+    RUN chmod +x ./kubectl-kuttl
+    RUN mv ./kubectl-kuttl /usr/local/bin
+    RUN yum install -y skopeo
+  from: openstack-operator-ci-build-deploy
+  to: openstack-operator-ci-build-deploy-kuttl
```
Comment (cescgina) [inline ci-operator/config/openstack-k8s-operators/horizon-operator/openstack-k8s-operators-horizon-operator-main.yaml:20]: The job fails becuase the `openstack-operator-ci-build-deploy` job and images are not configured for this repo.

Comment (jparoly) [reply ci-operator/config/openstack-k8s-operators/horizon-operator/openstack-k8s-operators-horizon-operator-main.yaml:20]: Is there a ticket for that implementation? Was it missed?  Can I add it?

Comment (cescgina) [reply ci-operator/config/openstack-k8s-operators/horizon-operator/openstack-k8s-operators-horizon-operator-main.yaml:20]: I don't think so, AFAIK nobody worked much in horizon-operator CI. You can add it in this patch, taking this one as an example: https://github.com/openshift/release/pull/38395/files

Comment (jparoly) [reply ci-operator/config/openstack-k8s-operators/horizon-operator/openstack-k8s-operators-horizon-operator-main.yaml:45]: done, please let me know if it looks correct.

Comment (jparoly) [reply ci-operator/config/openstack-k8s-operators/horizon-operator/openstack-k8s-operators-horizon-operator-main.yaml:45]: looks ready to merge just needs the lgtm tag  @bshephar

**Comment by cescgina** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-horizon-operator-main-horizon-operator-build-deploy-kuttl

**Comment by jparoly** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-horizon-operator-main-horizon-operator-build-deploy-kuttl

**Comment by jparoly** [comment]
the horizon kuttl tests should be passing now since this https://github.com/openstack-k8s-operators/install_yamls/pull/282 has merged no?

**Comment by cescgina** [comment]
Job is failing in the build stage due to missing modules in go.sum. This PR seems to fix the problem: https://github.com/openstack-k8s-operators/horizon-operator/pull/120

**Comment by cescgina** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-horizon-operator-main-horizon-operator-build-deploy-kuttl

**Comment by jparoly** [comment]
@bshephar can you please merge?

**Review by abays — APPROVED** [review]
/lgtm

**Review by bshephar — APPROVED** [review]
Looks good to me.

**Pull request merged**